### PR TITLE
Publish v4.0.0

### PR DESCRIPTION
**Pull request by surajeaton**
Fixes # Publish v4.0.0

#### Changes proposed in this Pull Request:

- Publish v4.0.0

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by huayunh** [comment]
Self Registration, Create Password Screen, text field too tight on mobile

Only on mobile. The desktop side looks fine.

image

The same applies to the reset password mobile screen

**Comment by JeffGreiner-eaton** [comment]
We have a unit test failing on forgot password screen. Passing in showSuccessScreen={false} isn't currently working. 

image

in the example if I pass in showSuccessScreen={false} nothing happens with onNext


https://github.com/etn-ccis/blui-react-workflows/assets/119693939/70d8dece-9892-4b77-806f-f3abbd163a95

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ */
+
+export const ContactSupportScreen: React.FC<ContactSupportScreenProps> = (props) => {
+    const { t } = useTranslation();
+    const { navigate, routeConfig } = useAuthContext();
+
+    const {
+        icon = <ChatBubbleOutline color={'primary'} sx={{ fontSize: 70 }} />,
+        emailSupportTitle = t('bluiAuth:CONTACT_SUPPORT.GENERAL_QUESTIONS'),
+        emailSupportContent,
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:36]: the emailSupportContent should have translation

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        phoneSupportTitle = t('bluiAuth:CONTACT_SUPPORT.EMERGENCY_SUPPORT'),
+        phoneSupportContent,
```
Comment (JeffGreiner-eaton) [inline login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx:38]: the phoneSupportContent should have translation

File: login-workflow/docs/customization.md [hunk 3]
```diff
@@ -1,119 +1,100 @@
...
 
-    // custom error message
-+   throw new Error('My Custom Login Error');
-    ...
+For information about handling errors in your application, refer to our [Error Management](./error-management.md) documentation.
 
-},
-```
+## Components
+For more details on customizing our components, refer to our [Components](./components/components.md) documentation.
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/customization.md:97]: should this link to .components/README.md

File: login-workflow/docs/customization.md [hunk 4]
```diff
@@ -1,119 +1,100 @@
...
+For information about handling errors in your application, refer to our [Error Management](./error-management.md) documentation.
 
-},
-```
+## Components
+For more details on customizing our components, refer to our [Components](./components/components.md) documentation. 
 
-For more details, read the [full API details](https://github.com/brightlayer-ui/react-auth-shared/blob/master/docs/API.md#loginerrordisplayconfig)
+## Screens
+For more details on customizing our screens, refer to our [Screens](./screens/screens.md) documentation.
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/customization.md:100]: should this link to should this link to .screens/README.md

File: login-workflow/docs/components/workflow-card.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+| onPrevious |  `() => void` | A function that is called when the previous button is clicked. |  |
+| showNext | `boolean` | Indicates whether the next button should be displayed | `true` |
+| canGoNext | `boolean \| (() => boolean)` | A boolean or function that indicates whether the next button should be enabled. | `true` |
+| nextLabel | `string` | The label on the next button |  |
+| onNext |  `() => void` | A function that is called when the next button is clicked. |  |
+| currentStep | `number` | Indicates the current step in a multi-step workflow | `0` |
+| totalSteps | `number` | Indicates the total number of steps in the multi-step workflow | `0` |
+| fullWidthButton | `boolean` | If true, buttons will expand to the full width of the panel (useful if you only have one button configured) | `false` |
+
+The properties of the underlying MUI [CardActions](https://mui.com/components/card-actions/) component are also available.
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/components/workflow-card.md:74]: this link is dead for me. should this be https://mui.com/material-ui/api/card-actions/ or https://mui.com/material-ui/react-card/

File: login-workflow/docs/components/change-password-dialog.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+| slots                         | `ChangePasswordDialogSlots`         | Components to use in place of the defaults. See [ChangePasswordDialogSlots](#changepassworddialogslots)                         |                                                               |
+| slotProps                     | `ChangePasswordDialogSlotsProps`    | Props to pass to the custom slot components. See [ChangePasswordDialogSlotsProps](#changepassworddialogslotsprops)              |                                                               |
+
+### ChangePasswordDialogSlots
+
+| Key           | Type                                         | Description                                                                    |
+| ------------- | -------------------------------------------- | ------------------------------------------------------------------------------ |
+| SuccessScreen | `(props: SuccessScreenProps) => JSX.Element` | A custom success screen component to render. See [SuccessScreen](./success.md) |
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/components/change-password-dialog.md:44]: link needs updated to screens/success

File: login-workflow/docs/components/change-password-dialog.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### ChangePasswordDialogSlotsProps
+
+| Key           | Type                 | Description                                                                             |
+| ------------- | -------------------- | --------------------------------------------------------------------------------------- |
+| SuccessScreen | `SuccessScreenProps` | Props to pass to the custom success screen component. See [SuccessScreen](./success.md) |
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/components/change-password-dialog.md:50]: link needs updated to screens/success

File: login-workflow/docs/customization.md (hunk 3 above)
Comment (JeffGreiner-eaton) [reply login-workflow/docs/customization.md:97]: https://github.com/etn-ccis/blui-react-workflows/pull/498

File: login-workflow/docs/customization.md (hunk 4 above)
Comment (JeffGreiner-eaton) [reply login-workflow/docs/customization.md:100]: https://github.com/etn-ccis/blui-react-workflows/pull/498

File: login-workflow/docs/components/change-password-dialog.md (hunk 6 above)
Comment (JeffGreiner-eaton) [reply login-workflow/docs/components/change-password-dialog.md:44]: https://github.com/etn-ccis/blui-react-workflows/pull/498

File: login-workflow/docs/components/change-password-dialog.md (hunk 7 above)
Comment (JeffGreiner-eaton) [reply login-workflow/docs/components/change-password-dialog.md:50]: https://github.com/etn-ccis/blui-react-workflows/pull/498

**Comment by ektaghag-eaton** [comment]
> We have a unit test failing on forgot password screen. Passing in showSuccessScreen={false} isn't currently working.
> 
> image
> 
> in the example if I pass in showSuccessScreen={false} nothing happens with onNext
> 
>  forgotPassword.mov

Fixed https://github.com/etn-ccis/blui-react-workflows/pull/500, @JeffGreiner-eaton please review.

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx (hunk 2 above)
Comment (ektaghag-eaton) [reply login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx]: Fixed https://github.com/etn-ccis/blui-react-workflows/pull/497

File: login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx (hunk 1 above)
Comment (ektaghag-eaton) [reply login-workflow/src/screens/ContactScreen/ContactSupportScreen.tsx]: Fixed https://github.com/etn-ccis/blui-react-workflows/pull/497

**Comment by JeffGreiner-eaton** [comment]
The ChangePasswordDialog test fails on initial test run and passes on the 2nd test run. Currently on a fresh clone I'm noticing issues with open prop.

image

**Comment by huayunh** [comment]
Let's have https://github.com/etn-ccis/blui-react-workflows/pull/502 merged in first

**Review by ektaghag-eaton — APPROVED** [review]

**Review by daileytj — APPROVED** [review]

**Pull request merged**